app/assets/stylesheets/_variables.scss

```diff
@@ -0,0 +1 @@
+$color-orange-light: #F89E54;
```

Comment:
Just spotted we now have a variables and a settings file doing the same thing. We should merge them into `variables.scss`